src/supermarket/spec/models/curry/repository_spec.rb

```diff
@@ -22,4 +22,22 @@
     expect(repository.owner).to eql('chef')
     expect(repository.name).to eql('supermarket')
   end
+
+  describe '#subscribe!' do
+    let(:expected_fqdn) { 'bunniesareadorable' }
+
+    before do
+      ENV['FQDN'] = expected_fqdn
+    end
```

Comment:
It seems to set it back, since it is effectively in a before each block, rather than a before all block (before all is where we would need to unset it).  It *seems* like it resets ENV['FQDN'] back to localhost, but I will confirm that.